idaes/core/tests/test_control_volume_base.py

```diff
@@ -48,7 +48,7 @@ def test_energy_balance_type():
 
     # Test that error is raised when given non-member
     with pytest.raises(AttributeError):
-        EnergyBalanceType.foo
+        EnergyBalanceType.foo  # pylint: disable=no-member
```

Comment:
I think the better response here would be to find a better way to test the `Enum`, rather than turning off pylint functionality. One could argue there isn't really need to test that an `Enum` returns an `AttributeError` if given a non-member. - we should just check that it has the expected members and be done.

Reply:
I see your point, but now I realize that this fix can probably be disregarded as we're (purposefully) not running pylint on test files. I tested it at some point, and it turned out to be lots of headaches (at least 5x runtime and lots of errors, most likely because of pylint) for very little gain (at least at the level of catching runtime errors not covered by tests, as tests are by definition run during... well, test runs).